### ISSUE TITLE
NY: Bruckner Interchange

### ISSUE THREAD

**Issue by yakra** [TravelMapping/HighwayData#252]
I-278 isn't connected to I-95 & pals at the Bruckner Interchange.
See the graph in mapview.php:
http://tm.teresco.org/hbtest/mapview.php?rg=NY&u=yakra
I-95 and especially I-278 could use some cleanup in the area; this rabbit hole gets pretty deep pretty fast, though I've mostly wrapped my head around it.
While I'm in there I'll see what else needs attention.

**Comment by yakra** [comment]
**I-278:**
Wacky Hijinks:
* Widening and reconstruction of the Staten Island Expressway, with reconfiguration of exits where things are already reminiscent of Texas.
Far Wackier Hijinks:
* An exit renumbering happened that I wasn't hip to, from NY25 north. Lots of newsworthy new/old label conflicts affect every existing waypoint from NY25A to the Bronx River Pkwy inclusive. Updates text should be interesting; this will need discussion.

**And now, the terminus:**
I've added Exit 6A to I-95, at the western split of the Bruckner Interchange, near the present I-278 endpoint; this point had been missing. Rather than just hook up I-278 to that and call it a day, I felt it best to continue the route east to its actual connection with I-95 at the meat of the interchange in the east. 
(The existing waypoint has been relabeled 54, tweaked slightly from I-95's coords to avoid a false-positive concurrency.)
I think it's most appropriate to move the existing I-95 +999 labels to the new endpoint: No need to break people's 100% clinches here, and plus, it's not really an extension of the highway but a more precise mapping of what was already there.
Should the extension still be listed in the Updates text along with all the other stuff? Yeah, probably. It's possible those who've traveled, say, to/from the Hutch might want to edit their 100% clinch down to a 98.7% clinch.

-----

Update thread on the old forum, with a complete list of changes & work in progress:
http://clinched.s2.bizhat.com/viewtopic.php?p=16732

I've committed files with my proposed changes; take a look:
https://github.com/yakra/HighwayData/blob/RobertMoses/hwy_data/NY/usai/ny.i095.wpt
https://github.com/yakra/HighwayData/blob/RobertMoses/hwy_data/NY/usai/ny.i278.wpt

**Comment by yakra** [comment]
I wanna make a pull request pretty soon.

1st item of business: Any objections to how I handled I-278's terminus?

**Comment by yakra** [comment]
No objections? Good! :D

Now, **the update text...** This style:
• (USA) Texas US 80: Moved waypoint TX34 from Virginia Street (now labeled TX34Bus) to Callie Street in Terrell.
...has been the "standard" that I've been gravitating towards.
But it's not very concise if there are eleven individually newsworthy relabelings. [I looked to the CHM hwyupdates for guidance/precedent and found a few examples from 2012.] (http://cmap.m-plex.com/docs/hwyupdates.php?y=2012)

So how about...
*(USA) New York I-278: Relabeled waypoints 38 to 41, 39 to 42, 40 to 43, 41 to 44, 42 to 45, 43 to 46, 44 to 47, 45 to 48, 46 to 50, 47 to 51, and 48 to 52.*
I'll make a couplefew more commits to take care of some low-hanging fruit then put in a pull request.
